osu.Game/Overlays/DashboardOverlay.cs

```diff
@@ -107,7 +106,10 @@ private void loadDisplay(Drawable display)
             LoadComponentAsync(display, loaded =>
             {
                 if (API.IsLoggedIn)
-                    loading.Hide();
+                {
+                    displayLoadedOperation?.Dispose();
+                    displayLoadedOperation = null;
```

Comment:
Right, but when user will be back online - it will trigger api state change which will trigger tab change which will reload the display once again. Or am I missing something?

Reply:
that's a really convoluted flow to go through if you ask me. i'm not sure being offline should keep the loading layer shown, it really complicates matters and makes this read buggy. should be a placeholder view if anything, but that's out of scope of this PR, so i'd just leave blank for now. we don't handle online well in many places as is anyway.